tests/Integrations/WordPress/V6_1/AutomatedLoginEventsTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\WordPress\V6_1;
+
+use DDTrace\Tests\Integrations\WordPress\AutomatedLoginEventsTestSuite;
+
+ /**
+ * @group appsec
+ */
+class AutomatedLoginEventsTest extends AutomatedLoginEventsTestSuite
```

Comment:
Automated login event tests were not tested in Wordpress 6.1. This makes all these tests to be executed